CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ## [Unreleased]
 
+### Added
+
+- Android: `AudioTrack.qualities.channelCount`, providing the channel count associated with the `AudioTrack`
```

Comment:
Channel count is on `AudioQuality` not on `AudioTrack` and `AudioTrack.qualities.channelCount` is not an actual API as `AudioTrack.qualities` is an array.

```suggestion
- Android: `AudioQuality.channelCount`, providing the channel count associated with `AudioQuality`.
```